multitenancy/pom.xml

```diff
@@ -62,5 +62,9 @@
             <version>${mockito.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Ah, bummer, I didn't think about the dependency that would move if we'd do this...
Let's move the Persistent Stream implementing class back to the auto-configuration module.

My apologies for sending you back and forth!